gfreshii-webapp/src/main/java/com/edbootcamp/view/IngredientViewImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.edbootcamp.view;
+
+import com.edbootcamp.api.views.IngredientView;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class IngredientViewImpl implements IngredientView{
```

Comment:
Both IngreientView, and IngredientImpl, should implement Ingredient ( I )

Reply:
I think my naming convention got you confused. when i split the app i left the interfaces alone... I've since renamed it all